src/Models/BaseElement.php

```diff
@@ -169,6 +170,15 @@ class BaseElement extends DataObject
      */
     private static $displays_title_in_template = true;
 
+    /**
+     * Determines whether a block should be indexable in search.
+     *
+     * @config
+     * @var boolean
+     * @see ElementalPageExtension::getElementsForSearch()
+     */
+    private static $is_indexable = true;
```

Comment:
```suggestion
    private static $search_indexable = true;
```
We probably want the word 'search' explicitly in the var name, so it's not confused with a database index.  Can also remove the `is_` prefix